app/javascript/application.js

```diff
@@ -0,0 +1,3 @@
+// Entry point for the build script in your package.json
+import "@hotwired/turbo-rails"
+import "./controllers"
```

Comment:
- - 
![Screenshot from 2024-01-16 09-14-17](https://github.com/Rifat8080/hello-rails/assets/107632780/1afa895b-004d-45f4-a71d-ba65c125fd49)

- [ ] Looking at the screenshot below, you can see i get an error when i tried runing the app with `./bin/dev`, You have not imported React, react-dom and the App component here and initialized the app. I'll suggest you a=carefully go through the lesson one more time and follow it diligently.